include/seqan3/alignment/matrix/detail/two_dimensional_matrix.hpp

```diff
@@ -254,6 +254,34 @@ class two_dimensional_matrix
     }
     //!\}
 
+    /*!\brief Explicit conversion to other matrix major order.
+     * \tparam other_value_t The target value type; must be assignable from `value_t`.
+     * \tparam other_allocator_t The allocator type used for the target matrix.
+     * \tparam other_order The other seqan3::detail::matrix_major_order; must not be the same as `order`.
```

Comment:
I think we agreed on intending if it's the same "group", e.g. `\tparam`, but I don't have strong feelings here.

Reply:
Did we fix this? I'd rather not like to do it.

Reply:
I'm not sure, but you can leave it as it is...

Reply:
I mean honestly I don't care much. But more often I am annoyed by fixing indentation because of name changes then that they are not aligned. At least I made my peace with it. But if it is ok for you I leave it as is.